components/post_view/post_list/post_list.jsx

```diff
@@ -0,0 +1,215 @@
+
+        for (let i = posts.length - 1; i >= 0; i--) {
+            const post = posts[i];
+
+            if (
+                post == null ||
+                post.type === PostTypes.EPHEMERAL_ADD_TO_CHANNEL ||
+                isUserActivityPost(post.type)
+            ) {
+                continue;
+            }
+
+            const postCtl = (
+                <Post
+                    ref={post.id}
+                    key={'post ' + (post.id || post.pending_post_id)}
+                    post={post}
+                    lastPostCount={(i >= 0 && i < Constants.TEST_ID_COUNT) ? i : -1}
+                    getPostList={this.getPostList}
+                />
+            );
+
+            const currentPostDay = Utils.getDateForUnixTicks(post.create_at);
+            if (currentPostDay.toDateString() !== previousPostDay.toDateString()) {
+                postCtls.push(
+                    <DateSeparator
+                        key={currentPostDay.getTime()}
+                        date={currentPostDay}
+                    />
+                );
+            }
+
+            const isNotCurrentUser = post.user_id !== currentUserId || isFromWebhook(post);
+            if (isNotCurrentUser &&
+                    lastViewed !== 0 &&
+                    post.create_at > lastViewed &&
+                    !Utils.isPostEphemeral(post) &&
+                    !renderedLastViewed) {
+                renderedLastViewed = true;
+
+                // Temporary fix to solve ie11 rendering issue
+                let newSeparatorId = '';
+                if (!UserAgent.isInternetExplorer()) {
+                    newSeparatorId = 'new_message_' + post.id;
+                }
+                postCtls.push(
+                    <div
+                        id={newSeparatorId}
+                        key='unviewed'
+                        ref='newMessageSeparator'
+                        className='new-separator'
+                    >
+                        <hr
+                            className='separator__hr'
+                        />
+                        <div className='separator__text'>
+                            <FormattedMessage
+                                id='posts_view.newMsg'
+                                defaultMessage='New Messages'
+                            />
+                        </div>
+                    </div>
+                );
+            }
+
+            postCtls.push(postCtl);
+            previousPostDay = currentPostDay;
+        }
+
+        return postCtls;
+    }
+
+    getPostList = () => {
+        return this.refs.postlist;
+    }
+
+    loadOlderPosts = () => {
+        const oldestPost = this.props.posts[this.props.posts.length - 1];
+        let oldestPostId = oldestPost.id;
+        if (oldestPost.type === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
+            oldestPostId = oldestPost.system_post_ids[0];
+        }
+        this.props.actions.loadOlderPosts(oldestPostId);
```

Comment:
Couldn't the PostListWrapper do this in the method it passes as `loadOlderPosts`?